MovieMatcher/Views/blue.xaml.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
```

Comment:
Dit is niet een erg beschrijvende naam. Heb je hier een betere naam voor?

Reply:
Na overleg: we verwachten dat het automatisch aanpast bij de merge met de "zoeken" branch